src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -3174,4 +3267,693 @@ export class PostgresCohortMembersService {
+        // Log email success
+        ShortlistingLogger.logRejectionEmailSuccess({
+          dateTime: new Date().toISOString(),
+          userId: userData.userId,
+          email: userData.email,
+          shortlistedStatus: 'rejected',
+          cohortId: cohortId,
+        });
+      }
+    } catch (error) {
+      // Re-throw the error to be handled by the calling method
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)